cypress/e2e/cloud/userProfile.ts

```diff
@@ -0,0 +1,507 @@
+              .contains('Veganomicon')
+              .should('be.visible')
+          })
+
+          it('changes the default account using the dropdown', () => {
+            cy.intercept({
+              path: '/api/v2/quartz/accounts/default',
+              method: 'PUT',
+            }).as('putQuartzDefaultAccount')
+
+            cy.getByTestID('user-profile--change-account-dropdown').click()
+            cy.getByTestID('user-profile--change-account-dropdown')
+              .getByTestID('dropdown-item')
+              .contains('Influx')
+              .click()
+
+            cy.getByTestID('user-profile--save-button').click()
+
+            cy.wait('@putQuartzDefaultAccount').then(() => {
+              // In cypress, it may :look: as though notifications were not generated, as this test closes them immediately.
+              cy.getByTestID('notification-success')
+                .contains('Saved changes to your profile.')
+                .should('exist')
+                .then(() => {
+                  cy.getByTestID('user-profile--change-account-dropdown')
+                    .contains('Influx')
+                    .should('be.visible')
+
+                  cy.getByTestID('user-profile--save-button').should(
+                    'have.attr',
+                    'disabled'
+                  )
+
+                  cy.getByTestID('notification-success--dismiss').click({
+                    force: true,
+                  })
+                })
+            })
+          })
+
+          it('changes the default account using the typeahead', () => {
+            cy.intercept({
+              path: '/api/v2/quartz/accounts/default',
+              method: 'PUT',
+            }).as('putQuartzDefaultAccount')
+
+            cy.getByTestID('user-profile--change-account-dropdown').click()
+
+            cy.getByTestID('user-profile--change-account-dropdown')
+              .getByTestID('input-field')
+              .type('Vega')
+              .then(() => {
+                cy.getByTestID('user-profile--change-account-dropdown')
+                  .getByTestID('dropdown-item')
+                  .contains('Veganomicon')
+                  .click()
+              })
+
+            cy.getByTestID('user-profile--change-account-header').click()
+
+            cy.getByTestID('user-profile--save-button').click()
+
+            cy.wait('@putQuartzDefaultAccount').then(() => {
+              cy.getByTestID('user-profile--change-account-dropdown')
+                .contains('Veganomicon')
+                .should('be.visible')
+              cy.getByTestID('notification-success')
+                .contains('Saved changes to your profile.')
+                .should('exist')
+
+              cy.getByTestID('user-profile--save-button').should(
+                'have.attr',
+                'disabled'
+              )
+
+              cy.getByTestID('notification-success--dismiss').click({
+                force: true,
```

Comment:
After some testing, `force` appears to be necessary to successfully dismiss [x] notifications.